src/main/webapp/WEB-INF/tags/generic.tag

```diff
@@ -33,7 +33,6 @@
     <meta name="theme-color" content="#75C8EC" /> <!-- Android Chrome mobile browser tab color -->
     <meta http-equiv="pragma" content="no-cache" />
 
-    <spring:eval expression="@externalServicesConfig.getAnalyticsCode()"/>
```

Comment:
Let's keep this line too in case we need to inject some JS later. Just ensure the CI/CD variables have been updated for prod.